src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -529,7 +527,7 @@ public int Main(string[] args)
 
         private Reports CreateReports(bool verbose, bool quiet)
         {
-            var useConsoleColor = _runtimeEnv.OperatingSystem == "Windows";
+            var useConsoleColor = false;
```

Comment:
Oh ya, well the AnsiConsole will be changed in another PR before this goes out, so I didn't change this.